doc/getting-started/apl-nuc.rst

```diff
@@ -364,6 +364,46 @@ For hugeTLB to work, you'll need to reserve huge pages:
      by using a 2M page, but make sure your huge page reservation size is
      large enough for your usage.
 
+USB Device Sharing
+==========================================
+
+The ACRN hypervisor is able to support USB device sharing
```

Comment:
Some grammar, sentence structure, and reST syntax fixes:

    The ACRN hypervisor supports USB device sharing.  Suppose you have
    two keyboards and mice connected to your device, one keyboard and 
    mouse set for the SOS, and the other set for the UOS.  

    1. Boot the SOS and plug in the two keyboards and two mice
       into four available USB ports on the device (via a USB hub
       if required).

    #. Run ``dmsg`` to find the kernel messages logging the enumeration
       of the connected keyboards and mice.  For example::

          keyboard 1#: USB/1-3/1-3
          mouse 1#: USB/1-4/1-4
          keyboard 2#: USB/1-2/1-2
          mouse 2#: USB/1-1/1-1

       You'll need this device information in the next step.

    #. We can now assign keyboard 1# and mouse 1# to the SOS, and
       keyboard 2# and mouse 2# to the UOS.  Use a text editor to modify
       ``/usr/share/acrn/samples/nuc/launch_uos.sh`` and add the line (using
       the keyboard and mouse identified in your dmesg output)::

          -s 9, xhci, 1-1:1-2 \

       Save the file, exit the editor, and run the ``sync`` command to ensure
       any pending write buffers are written to disk.